css/layout.css

```diff
@@ -23,12 +23,22 @@
     max-width: 900px;
   }
 
-  .column-full,
   .column-half {
-    padding: 0 0.75rem;
+    width: 50%;
   }
 
-  .column-half {
+  .row-one {
+    display: flex;
+  }
+
+  .row1-col2 {
     width: 50%;
+    margin: 0 0 0 15px;
+  }
+
+  .image {
+    width: 50%;
```

Comment:
You should move this to `styles.css`. The only CSS that I would apply here would be your container / row / col classes or utility classes